homeassistant/components/mill/climate.py

```diff
@@ -66,6 +66,11 @@
         dev.append(MillHeater(heater, mill_data_connection))
     async_add_entities(dev)
 
+    dev = []
```

Comment:
Remove line 67 and 69

Reply:
Sure, just me beeing lazy with cut and pase.

This is my very first dive into github (besides using componets I find there). I hope I've edited the code in the right place.
Removed lines 67, 68 and 69.